src/app/modules/core/advertisement/components/advertisement-list/card/card-extra/review/review.component.ts

```diff
@@ -1,26 +1,14 @@
-import { Component, OnInit } from '@angular/core'
+import { Component, Input, OnInit } from '@angular/core'
+
+import { MockAdvertisement } from './../../../../../mock-advertisement/mock-advertisement'
 
 @Component({
   selector: 'cav-review',
   templateUrl: './review.component.html',
   styleUrls: ['./review.component.scss'],
 })
 export class ReviewComponent implements OnInit {
-  reviews = [
-    {
-      title: 'Good',
-      tenantId: 3,
-      description: 'Very spacious and accessible house',
-      vote: 4.5,
-    },
-    {
-      title: 'Not great',
-      tenantId: 4,
-      description: 'The hostel was very very chaotic ',
-      vote: 1.5,
-    },
-  ]
-
+  @Input() advertisement: MockAdvertisement
```

Comment:
Deve prendere un input di tipo review and non MockAdvertisement.

Reply:
fixed